public/javascripts/script.js

```diff
@@ -0,0 +1,13 @@
+
+var socket = io();
+$(function () {
+  $('form').submit(function(){
+    socket.emit('chat message', $('#messageInput').val());
+    $('#messageInput').val('');
+    return false;
```

Comment:
It's more explicit to pass 'event' on line 4 and inside the function do 'event.preventDefault()', instead of returning false.